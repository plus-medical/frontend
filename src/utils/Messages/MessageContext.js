import React, { useState, createContext } from 'react'
const MessageContext = createContext()

export default function MessageProvider ({ children }) {
  const [message, setMessage] = useState({
    message: '',
    type: ''
  })

  return (
    <MessageContext.Provider
      value={{
        message,
        setMessage
      }}
    >
      {children}
    </MessageContext.Provider>
  )
}

export { MessageContext, MessageProvider }
