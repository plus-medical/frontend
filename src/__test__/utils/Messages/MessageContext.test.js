/* eslint-disable no-undef */
import React from 'react'
import { create } from 'react-test-renderer'
import RouterMock from '../../../__mocks__/RoutesMocks'
import MessageContext from '../../../utils/Messages/MessageContext'

describe('<MessageContext />', () => {
  test('Review Snapshot in MessageContext component', () => {
    const messageContext = create(
      <RouterMock>
        <MessageContext />
      </RouterMock>
    )
    expect(messageContext.toJSON()).toMatchSnapshot()
  })
})
