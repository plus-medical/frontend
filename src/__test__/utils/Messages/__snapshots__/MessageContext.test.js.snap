// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MessageContext /> Review Snapshot in MessageContext component 1`] = `null`;
